releasenotes/notes/use-gauge-metrics-in-rm-61ae620e83b95740.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    runtime_metrics: Updates runtime metrics to submit guage metrics instead of distribution metrics.
```

Comment:
nit: small typo:
```suggestion
    runtime_metrics: Updates runtime metrics to submit gauge metrics instead of distribution metrics.
```

Reply:
If the new plan is to change the runtime metric names, then this release note should also be updated as it's a fairly large change.